photoeditor/src/main/java/ja/burhanrashid52/photoeditor/Sticker.java

```diff
@@ -0,0 +1,60 @@
+package ja.burhanrashid52.photoeditor;
+
+import android.graphics.Bitmap;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+/**
+ * Created by Burhanuddin Rashid on 14/05/21.
+ *
+ * @author <https://github.com/burhanrashid52>
+ */
+class Sticker extends Graphic {
+
+    private final MultiTouchListener mMultiTouchListener;
+    private final ViewGroup mPhotoEditorView;
+    private final PhotoEditorViewState mViewState;
+    private ImageView imageView;
+
+    public Sticker(ViewGroup photoEditorView,
+                   MultiTouchListener multiTouchListener,
+                   PhotoEditorViewState viewState,
+                   GraphicManager graphicManager
+    ) {
+        super(photoEditorView.getContext(), graphicManager);
+        mPhotoEditorView = photoEditorView;
+        mViewState = viewState;
+        mMultiTouchListener = multiTouchListener;
+    }
+
+    void buildView(Bitmap desiredImage) {
+
+        imageView.setImageBitmap(desiredImage);
+
+        MultiTouchListener.OnGestureControl onGestureControl = buildGestureController(mPhotoEditorView, mViewState);
+        mMultiTouchListener.setOnGestureControl(onGestureControl);
+
+        View rootView = getRootView();
+        rootView.setOnTouchListener(mMultiTouchListener);
+        clearHelperBox();
+        addViewToParent();
+        mViewState.setCurrentSelectedView(rootView);
+    }
+
+
+    @Override
+    ViewType getViewType() {
+        return ViewType.IMAGE;
```

Comment:
Potentially dumb question: Any reason we're not changing this to be `ViewType.STICKER`? Out of scope? Maybe this is exposed to the user?

Reply:
Yes, It's exposed to the user in `OnPhotoEditorListener` and hence breaks the client code.